### ISSUE TITLE
Introduce XO

### ISSUE THREAD

**Issue by shinnn** [imagemin/meta#10]
https://github.com/sindresorhus/xo

**Comment by sindresorhus** [comment]
Sure, but let's give it a few weeks to mature.

**Comment by shinnn** [comment]
:ok:

**Comment by kevva** [comment]
Yup, I'm just going to use it in some of my private projects for now and see what issues I (and others) hit until switching fully.

**Comment by sindresorhus** [comment]
I think it's time. We should also convert to AVA for tests and some repos are using outdated AVA version which also needs upgrading. @shinnn Interested in doing this?

**Comment by shinnn** [comment]
- [x] https://github.com/imagemin/api
- [ ] https://github.com/imagemin/imagemin.io
- [ ] https://github.com/imagemin/imagemin-app
- [x] https://github.com/imagemin/imagemin-optipng
- [x] https://github.com/imagemin/jpegtran-bin
- [x] https://github.com/imagemin/zopflipng-bin
- [x] https://github.com/imagemin/imagemin-gifsicle
- [ ] https://github.com/imagemin/advpng-bin
- [x] https://github.com/imagemin/pngcrush-bin
- [x] https://github.com/imagemin/imagemin-zopfli
- [x] https://github.com/imagemin/imagemin-jpeg-recompress
- [x] https://github.com/imagemin/imagemin-pngcrush
- [ ] https://github.com/imagemin/jpeg-recompress-bin
- [ ] https://github.com/imagemin/imagemin-pngout
- [x] https://github.com/imagemin/api

@sindresorhus Will do.

**Comment by sindresorhus** [comment]
@shinnn Did you finish this? Can this be closed now?

XO has changed a lot since this though, so we should probably run the latest version on all the repos and fix issues.

**Comment by sindresorhus** [comment]
Ping @shinnn :)
